internal/kuberesource/parts.go

```diff
@@ -173,6 +173,41 @@ func (p *PortForwarderConfig) WithForwardTarget(host string, port int32) *PortFo
 	return p
 }
 
+// MultiplePortsForwarder constructs a port forwarder pod for multiple ports.
+func MultiplePortsForwarder(name, namespace string) *PortForwarderConfig {
+	name = "port-forwarder-" + name
+
+	p := Pod(name, namespace).
+		WithLabels(map[string]string{"app.kubernetes.io/name": name}).
+		WithSpec(PodSpec().
+			WithContainers(
+				Container().
+					WithName("port-forwarder").
+					WithImage("ghcr.io/edgelesssys/contrast/port-forwarder:latest").
+					WithCommand("/bin/bash", "-c", "echo Starting port-forward with socat; for port in ${LISTEN_PORTS}; do exec socat -d -d TCP-LISTEN:$port,fork TCP:${FORWARD_HOST}:$port & done; while true; do sleep 10; done;").
```

Comment:
```suggestion
					WithCommand("/bin/bash", "-c", "echo Starting port-forward with socat; for port in ${LISTEN_PORTS}; do exec socat -d -d TCP-LISTEN:$port,fork TCP:${FORWARD_HOST}:$port & done; wait").
```

Reply:
I did try `wait` initially, however this does not work because the processes launched in the loop dont live in the same shell as `wait` does.

Reply:
Because of the exec?

Reply:
Tried it again to make sure I wasn't wrong, `wait` works now for some reason. I assume that I accidentally wrote added wait in the wrong place when I was trying it out initially.